src/omotes_sdk/internal/common/broker_interface.py

```diff
@@ -17,6 +18,7 @@
     AbstractIncomingMessage,
     AbstractExchange,
 )
+from pamqp.common import Arguments
```

Comment:
We should add a dependency on pamqp in `pyproject.toml` if we reference it ourselves. We cannot reference dependencies of dependencies and assume they will remain indefinitely as they are controlled by our direct dependency.

Reply:
Added in 2ddfcc981500a4940a0317fbc69ea854c635dfeb